src/main/resources/liquibase/crop_changelog/16_4_0.xml

```diff
@@ -0,0 +1,225 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+	<changeSet author="mderamos" id="v16.4.0-1">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">
+				SELECT
```

Comment:
I think it's more intuitive and easier to understand to check if there are any records that have **not null** germplasm_uuid values.
```
<sqlCheck expectedResult="1">
				SELECT COUNT(*) > 0
				FROM germplsm
				WHERE germplsm_uuid IS NOT NULL;
			</sqlCheck>
```